catalog/app/utils/spreadsheets/mocks/bilbo.csv

```diff
@@ -0,0 +1,2 @@
+Name,Age,Male,Parts,Date,Fingers
```

Comment:
Probably, `.csv` + `.fods` should stay, and other formats should be removed

Reply:
why?

Reply:
Because, `.csv` and `.fods` are text-based formats and git-diff friendly. And I don't interfere in parsing different spreadsheets formats, it's the `xlsx` business.

Reply:
true, but i think it helps if we could spot inconsistencies in handling of different file formats